app/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -322,7 +322,7 @@ export function mockExperiment<
       isEnrollmentPaused: true,
       enrollmentEndDate: null,
       recipeJson:
-        '{"schemaVersion": "1.5.0", "slug": "pre-emptive-zero-tolerance-data-warehouse", "id": "pre-emptive-zero-tolerance-data-warehouse", "arguments": {}, "application": "", "appName": "firefox_ios", "appId": "", "channel": "", "userFacingName": "Pre-emptive zero tolerance data-warehouse", "userFacingDescription": "Analysis art mean sort serve stuff. Scene alone current television up write company. Without admit she occur total generation by mother. Environmental remember account huge drive policy play strong.", "isEnrollmentPaused": false, "bucketConfig": null, "probeSets": [], "outcomes": [{"slug": "example_config", "priority": "primary"}, {"slug": "newtab_visibility", "priority": "primary"}, {"slug": "picture_in_picture", "priority": "secondary"}], "branches": [{"slug": "horizontal-well-modulated-conglomeration", "ratio": 1, "feature": {"featureId": "decentralized-solution-oriented-neural-net", "enabled": true, "value": {"trouble-left-back": "west-receive"}}}, {"slug": "fully-configurable-context-sensitive-local-area-network", "ratio": 1, "feature": {"featureId": "decentralized-solution-oriented-neural-net", "enabled": true, "value": {"financial-school": "peace-light-might"}}}], "targeting": "localeLanguageCode == \'en\' && ((isFirstStartup && !(\'trailhead.firstrun.didSeeAboutWelcome\'|preferenceValue)) || experiment.slug in activeExperiments)", "startDate": null, "endDate": null, "proposedDuration": 60, "proposedEnrollment": 45, "referenceBranch": "horizontal-well-modulated-conglomeration", "featureIds": ["decentralized-solution-oriented-neural-net"]}',
+        '{\n  "appId": "firefox-desktop",\n  "appName": "firefox_desktop",\n  "application": "firefox-desktop",\n  "arguments": {},\n  "branches": [\n    {\n       "feature": {\n       "enabled": false,\n       "featureId": null,\n       "value": {}\n     },\n     "ratio": 1,\n     "slug": "control"\n    }\n  ],\n  "bucketConfig": null,\n  "channel": "",\n  "endDate": null,\n  "featureIds": [],\n  "id": "pre-emptive-zero-tolerance-data-warehouse",\n  "isEnrollmentPa… "control",\n  "schemaVersion": "1.5.0",\n  "slug": "pre-emptive-zero-tolerance-data-warehouse",\n  "startDate": null,\n  "targeting": "\'app.shield.optoutstudies.enabled\'|preferenceValue",\n  "userFacingDescription": "Analysis art mean sort serve stuff. Scene alone current television up write company. Without admit she occur total generation by mother. Environmental remember account huge drive policy play strong.",\n  "userFacingName": "Pre-emptive zero tolerance data-warehouse"\n}',
```

Comment:
My guts are a bit scared around here...

```js
,\n  "isEnrollmentPa… "control",\n  "schemaVersion": "1.5.0",\n
```

Not sure what is up with the ellipsis in the middle of `"isEnrollmentPa… "control"`.

Reply:
Huh, that's a great catch, it must have come from when I copied the response. o.O